lib/active_resource/base.rb

```diff
@@ -1096,23 +1123,12 @@ def find_every(options)
           params = options[:params]
           prefix_options, query_options = split_options(params)
 
-          response =
-            case from = options[:from]
-            when Symbol
-              get(from, params)
-            when String
-              path = "#{from}#{query_string(query_options)}"
-              format.decode(connection.get(path, headers).body)
-            else
-              path = collection_path(prefix_options, query_options)
-              format.decode(connection.get(path, headers).body)
-            end
-
-          instantiate_collection(response || [], query_options, prefix_options)
-        rescue ActiveResource::ResourceNotFound
-          # Swallowing ResourceNotFound exceptions and return nil - as per
-          # ActiveRecord.
-          nil
+          collection_parser.new([], options[:from]).tap do |parser|
```

Comment:
Network call now gets made within the `ActiveResource::Collection` class, so this is simply initializing it.